hadoop-ozone/dist/src/main/smoketest/debug/ozone-debug-ldb.robot

```diff
@@ -55,6 +55,14 @@ Test ozone debug ldb scan
                         Should contain          ${output}       testfile1
                         Should contain          ${output}       testfile2
                         Should contain          ${output}       testfile3
+    # test key is included with --with-keys
+    ${output1} =        Execute                 ozone debug ldb --db=/data/metadata/om.db scan --cf=keyTable | jq '."\/cli-debug-volume\/cli-debug-bucket\/testfile1"'
+    ${output2} =        Execute                 ozone debug ldb --db=/data/metadata/om.db scan --cf=keyTable --with-keys | jq '."\/cli-debug-volume\/cli-debug-bucket\/testfile1"'
```

Comment:
The third equivalent option to test here is `--with-keys=true`

Reply:
Thanks for the review, I have added this case too now